.circleci/config.yml

```diff
@@ -68,20 +68,18 @@ jobs:
     steps:
       - run-target:
           target: "pandas"
+  tardis:
+    executor: test-executor
+    steps:
+      - run-target:
+          target: "tardis"
 
 
 workflows:
   version: 2.1
   commit-integration-testing:
     jobs:
-      - umap
-      - librosa
-      - clifford
-      - awkward
-      - sparse
-      - fastparquet
-      - datashader
-      - pandas
+      - tardis
```

Comment:
We will wanto re-insert these, once the tardis testing is working well.

Reply:
Yes, they were removed right now so we could see where tardis was failing the pipeline without waiting for ~40 minutes per run as other tests would complete.

Reply:
Excellent, a smart choice! 😎